### ISSUE TITLE
Add Search

### ISSUE THREAD

**Issue by jaredwyee** [cagov/drought.ca.gov#71]
**What:**
Add Google Search functionality following design system pattern placement in upper right of page (see Cannabis), but styled consistent with Drought CSS and color scheme.

**Why:**
* Help users locate content as we start to flesh out what's on the drought site (more and more stuff!)
* Alternative ways to find information (accessibility angle as well)
* Design system consistency (Cannabis has search)

**Example:**
https://cannabis.ca.gov/
https://cannabis.ca.gov/serp/?q=testing

**Acceptance:**
* Drought site content is ingested/scrapped appropriately so that it can be returned as search results
* User able to input search string
** Obtain search results when there are results from the Drought site
** Obtain "no results" view when there are no results from the Drought site. Example: https://cannabis.ca.gov/serp/?q=blahblah
* With search implemented, ensure we have appropriate level instrumentation so we can see search terms look uped as part of site analytics. Helps determine trends. Maybe this needs to be a separate ticket and needs to involve ODI Insights team.

**Dependency**
Fixing menu before we can ship search.

**Comment by jaredwyee** [comment]
@xjensen can you start looking into this into next week? Happy to hammer our more implementation specifics as followup, but would be great if you could do an initial pass with Jim/Chach to start. Thanks!

**Comment by xjensen** [comment]
@jaredwyee, yes, can do. I'll start familiarizing myself with the challenge.

**Comment by abquirarte** [comment]
cc @adam-little in case there is design work here we want to consider.

**Comment by adam-little** [comment]
No additional design changes at this point. Just match Cannabis (except use Drought color).
